src/call/mod.rs

```diff
@@ -17,6 +17,9 @@ use crate::error::{Error, Result};
 use crate::grpc_sys::grpc_status_code::*;
 use crate::task::{self, BatchFuture, BatchType, CallTag, SpinLock};
 
+// By default buffers in `SinkBase` will be shrink to 4K size.
+const BUF_SHRINK_SIZE: usize = 4 * 1024;
```

Comment:
I worry 4k maybe a little small, we should do many benchmarks before merging into TiKV